projects/plugins/crm/modules/woo-sync/includes/class-woo-sync-background-sync-job.php

```diff
@@ -1705,6 +1725,39 @@ public function woocommerce_order_to_crm_objects(
 
 			}
 
+			// --- Process any present fee in the order --- //
+			$fees = $order->get_fees();
+
+			if ( is_array( $fees ) && count( $fees ) > 0 ) {
+				foreach ( $fees as $fee ) {
+					if ( $fee instanceof \WC_Order_Item_Fee ) {
+						$new_line_item = array(
+							'order'    => $order_post_id, // passed as parameter to this function
+							'currency' => $order_currency,
+							'quantity' => 1,
+							'price'    => $fee->get_amount( false ),
+							'fee'      => $fee->get_amount( false ),
+							'total'    => $fee->get_amount( false ),
+							'title'    => esc_html__( 'Fee', 'zero-bs-crm' ),
+							'desc'     => $fee->get_name(),
+							'tax'      => $fee->get_total_tax(),
+							'taxes'    => -1,
+							'shipping' => 0.0,
+						);
+
+						// Apply the same tax
+						if ( is_array( $item_tax_rate_ids ) && count( $item_tax_rate_ids ) > 0 ) {
+							$new_line_item['taxes'] = implode( ',', $item_tax_rate_ids );
+						}
```

Comment:
I'm a bit confused here...why would we give a line item a CSV of tax rate IDs instead of a specific tax rate ID?

Reply:
No answer here, legacy code. See line 1694 (old code).